controller/RestTestCenterUsers.php

```diff
@@ -206,25 +240,42 @@ public function get()
 
     /**
      * @return \oat\oatbox\user\User
-     * @throws \common_exception_BadRequest
      * @throws \common_exception_Error
-     * @throws \common_exception_MissingParameter
+     * @throws \common_exception_NotFound
+     * @throws \common_exception_RestApi
      */
     private function getUserFromRequest()
     {
-        $userUri = $this->getParameterFromRequest(self::PARAMETER_USER_URI);
-        return $this->getUserService()->getUserById($userUri);
+        try {
+            $userUri = $this->getParameterFromRequest(self::PARAMETER_USER_URI);
+        } catch (\common_exception_MissingParameter $e) {
+            throw new \common_exception_RestApi(__('Missed required parameter: `%s`', self::PARAMETER_USER_URI));
+        }
+        $user = $this->getUserService()->getUserById($userUri);
+        if (!$user || !$this->getResource($user->getIdentifier())->exists()) {
```

Comment:
Are you sure that will be compatible with KV implementations?

Reply:
A good point think about. For this API as well as for test center management form, only users stored in ontology storage are relevant. Lti users are not available in the interface and also not relevant here. Even if the LtiUserService would return a KvLtiUser this code will not break, it will just not find the user with KV identifier in the database. So I don't consider this should be changed